server/lib/src/schema.rs

```diff
@@ -591,7 +591,7 @@ pub trait SchemaTransaction {
                 a_schema.replicated.into() && !a_schema.phantom
             }
             None => {
-                warn!(
+                debug!(
```

Comment:
This should be a warning still, it shows that when we went to get an attribute, it wasn't in the schema at all, so something is wrong.